modules/tfm/zephyr/Kconfig

```diff
@@ -3,6 +3,7 @@
 #
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
+menu "TFM"
```

Comment:
why creating this menu when all settings inside the menu are promptless ?

Reply:
Left-over from refactoring. I had some configs in here earlier.

Will remove.